From: Yue Ma <yuem@codeaurora.org>
Date: Wed, 30 Jun 2021 17:22:34 -0700
Subject: [PATCH] qcacld-3.0: Add proper check to include qcom-iommu-util.h

qcom-iommu-util.h is a QCOM only file for MSM platforms, so only
include it when MSM_PLATFORM is defined to avoid build issues
from other non-QCOM platforms like X86.

Change-Id: I5f32fbe05abe4d7aff451031f0859f774ce1c507
CRs-fixed: 3721214
---
 core/cds/src/cds_api.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/cds/src/cds_api.c b/core/cds/src/cds_api.c
index d135759ac426..4fddb614889d 100644
--- a/core/cds/src/cds_api.c
+++ b/core/cds/src/cds_api.c
@@ -36,7 +36,7 @@
 #include "wlan_hdd_power.h"
 #include "wlan_hdd_tsf.h"
 #include <linux/vmalloc.h>
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 9, 0)) && defined(MSM_PLATFORM)
 #include <linux/qcom-iommu-util.h>
 #endif
 #include <scheduler_core.h>
